file_formats/cmx_format.mli

```diff
@@ -45,9 +45,13 @@ type generic_fns =
     send_fun: apply_fn list }
 
 type unit_infos =
-  { mutable ui_name: modname;             (* Name of unit implemented *)
-    mutable ui_symbol: string;            (* Prefix for symbols *)
-    mutable ui_defines: string list;      (* Unit and sub-units implemented *)
+  (* CR lmaurer: Consider renaming [ui_name], say to [ui_unit]. Code like
```

Comment:
Just do it, this is easy.

Reply:
Assuming we want to keep it a `CU.t` rather than a `CU.Name.t`?

Reply:
Done.